src/keyboard.c

```diff
@@ -203,17 +243,20 @@ keyboard_key_notify(struct wl_listener *listener, void *data)
 	struct wlr_keyboard *keyboard = &seat->keyboard_group->keyboard;
 	wlr_idle_notify_activity(seat->wlr_idle, seat->seat);
 
-	bool handled = false;
+	bool handled = handle_compositor_keybindings(listener, event);
 
-	/* ignore labwc keybindings if input is inhibited */
-	if (!seat->active_client_while_inhibited) {
-		handled = handle_compositor_keybindings(listener, event);
-	}
+	/*
+	 * Keys can be bound by C-A-F?? for TTY change, window cycling
+	 * (typically alt-tab, but could be any key/modifier), and user defind
+	 * keybinds
+	 */
+	wlr_log(WLR_DEBUG, "Number of pressed keys that were bound: %d\n",
+		key_state_nr_keys());
```

Comment:
Should we maybe put this within a ifdef to prevent spamming the debug log on keypress?
```c
#ifdef LAB_DEBUG_INPUT
[..]
#endif
```

Reply:
I've just deleted the commit that added that `wlr_log()`. For keyboard debugging purpose you are likely to need `fprintf()s` is several places, so on reflection, I don't see much point in keeping it.

Reply:
also works :)